mitodl/open-discussions
# Cc/styling tweaks

#### Pre-Flight checklist


#### What are the relevant tickets?
#3672 

#### What's this PR do?
Styling updates per Ferdi's feedback. See commit messages or https://mit.enterprise.slack.com/files/W010W1KTE73/F03SWRZ8S6L/https-discussions-rc-odl-mit-edu-infinite-fields-biology-.png

#### How should this be manually tested?
- Check that the pages work and look like they do below. Keep an eye out for any styling oddities.
- Check that the field page title links to the field page itself, similar to open-discussions channels

#### Screenshots (if appropriate)

**Notes:**
- Search page still needs styling, my edits there were fairly small.
- Regarding avatar colors, see https://github.com/mitodl/hq/discussions/194

<table>
    <thead></thead>
        <tr>
            <th></th>
            <th>Mobile</th>
            <th>Desktop</th>
        </tr>
    <tbody>
        <tr>
            <th>Home</th>
            <td>
                <img width="400" alt="home_mobile" src="https://user-images.githubusercontent.com/9010790/183095712-26284a90-1d54-424d-87b1-31338f1ddfa2.png">
            </td>
            <td>
                <img width="400" alt="home_wide" src="https://user-images.githubusercontent.com/9010790/183088541-07815dc7-a423-4ada-b5b9-7535fada29bf.png">
            </td>
        </tr>
        <tr>
            <th>View Field</th>
            <td>
                <img width="400" alt="field_mobile" src="https://user-images.githubusercontent.com/9010790/183092054-d4dd9ae9-01a7-41c7-b8cb-3eb85d7b9035.png">
            </td>
            <td>
                <p>⚠️ Note: there's a right-hand column with no content at the moment (for widgets), which is why it looks un-centered.</p>
                <img width="400" alt="field_wide" src="https://user-images.githubusercontent.com/9010790/183092047-6a672d26-7f67-4148-a31e-27e9b642975f.png">
            </td>
        </tr>
        <tr>
            <th>Manage Field</th>
            <td>
                <img width="400" alt="manage_mobile" src="https://user-images.githubusercontent.com/9010790/183088543-0517be49-abcf-4ce8-b3e2-598e6c2b5195.png">
            </td>
            <td>
                <img width="400" alt="manage_wide" src="https://user-images.githubusercontent.com/9010790/183088546-3887f01c-8f81-49b7-af36-303d8a53ee16.png">
            </td>
        </tr>
        <tr>
            <th>Search</th>
            <td>
                <img width="250" alt="search_mobile" src="https://user-images.githubusercontent.com/9010790/183094445-ccb97e7d-6c23-4d3c-9dd8-2d8cdcce76ca.png">
            </td>
            <td>
                <img width="400" alt="search_wide" src="https://user-images.githubusercontent.com/9010790/183094450-2f42e8e5-0ce6-42b3-9c2e-8c41264211e8.png">
            </td>
        </tr>
    </tbody>
</table>
